implementations/rust/ockam/ockam_api/src/nodes/service.rs

```diff
@@ -322,7 +328,7 @@ impl NodeManager {
                 self.set_authorities(req, dec).await?.to_vec()?
             }
             (Post, ["node", "credentials", "actions", "get"]) => {
-                self.get_credential(ctx, req, dec).await?.to_vec()?
+                self.get_credential(req, dec).await?.to_vec()?
```

Comment:
Why is this method called `get` if it's setting the credential and it's not returning anything?

Reply:
@adrianbenavides in this case it means that background node should go get a credential for itself

Reply:
Path also indicates that this is an action, not a getter

Reply:
ok, got it, thanks!